app/views/spree/orders/_bought.html.haml

```diff
@@ -3,7 +3,7 @@
     %td.toggle-bought{ colspan: 2, ng: { click: 'showBought=!showBought' } }
       %h5.brick
         %i{ ng: { class: "{ 'ofn-i_007-caret-right': !showBought, 'ofn-i_005-caret-down': showBought}"} }
-        = t(:orders_bought_items_notice, count: @order.finalised_line_items.sum(&:quantity))
+        = t(:orders_bought_items_notice, count: @order.finalised_line_items.to_a.sum(&:quantity))
```

Comment:
This can be `finalised_line_items.sum(:quantity)` right?

Reply:
tests do break with that, in terms of prod code may it would work but there are quite some tests that need this. That's why I went this way...